src/MarkdownTextInput.web.tsx

```diff
@@ -236,6 +239,33 @@ const MarkdownTextInput = React.forwardRef<MarkdownTextInput, MarkdownTextInputP
       [parser, parseText, processedMarkdownStyle],
     );
 
+    const handleFormatSelection = useCallback(
+      (target: MarkdownTextInputElement, parsedText: string, cursorPosition: number, formatCommand: string): ParseTextResult => {
+        if (!contentSelection.current || contentSelection.current.end - contentSelection.current.start < 1) {
+          return {
+            text: '',
+            cursorPosition: 0,
+          };
```

Comment:
When exactly will this case happen? If this case is not expected, wouldn't it be better to throw an error instead?

Reply:
I don't think it's possible to happen. Updated to throw an error.